src/scss/_main.scss

```diff
@@ -4,7 +4,7 @@ body {
   align-items: center;
   width: 100vw;
   min-height: 100vh;
-  background-image: url(images/background-image.png);
+  background-image: url(/images/background-image.png);
```

Comment:
/ potrzebny? ostatnio go usunąłem

Reply:
Też mi się wydaje, że "/" nie powinno zostać, bo to zostanie potraktowane jako ścieżka bezwzględna, a tutaj musimy użyć względnej, czyli albo bez niczego, albo "./".

Reply:
jak widzisz na mainie, bez '/' nie działa tło. Ścieżka bezwzględna tutaj jak najbardziej pasuje, ponieważ nasz folder images znajduje się w głównym katalogu. Logo też tak ustawiłem i dlatego już działa

Reply:
Ok, myślałem, że przez pomyłkę tak zrobiłeś, ale jeśli to intencjonalne i sprawdzone, to wporzo :)